_includes/footer-theme-wet-boew-en.html

```diff
@@ -1,59 +1,46 @@
-<footer role="contentinfo" id="wb-info" class="visible-sm visible-md visible-lg wb-navcurr">
-	<div class="container">
-		<nav role="navigation" class="row">
-			<h2>About this site</h2>
-			<section class="col-sm-3">
-				<h3>Contact us</h3>
-				<ul class="list-unstyled">
-					<li><a href="https://github.com/wet-boew/wet-boew/issues/new">Questions or comments?</a></li>
-				</ul>
-			</section>
-			<section class="col-sm-3">
-				<h3>About</h3>
-				<ul class="list-unstyled">
-					<li><a href="../../index-en.html#about">About the Web Experience Toolkit</a></li>
-					<li><a rel="external" href="http://www.tbs-sct.gc.ca/ws-nw/index-eng.asp">About the Web Standards</a></li>
-				</ul>
-			</section>
-			<section class="col-sm-3">
-				<h3>News</h3>
-				<ul class="list-unstyled">
-					<li><a href="https://github.com/wet-boew/wet-boew/pulse">Recent project activity</a></li>
-					<li><a href="https://github.com/wet-boew/wet-boew/graphs">Project statistics</a></li>
-				</ul>
-			</section>
-			<section class="col-sm-3">
-				<h3>Stay connected</h3>
-				<ul class="list-unstyled">
-					<li><a rel="external" href="https://twitter.com/WebExpToolkit">Twitter</a></li>
-				</ul>
-			</section>		</nav>
-	</div>
+<footer id="wb-info" class="visible-sm visible-md visible-lg wb-navcurr">
+<div class="container">
+<nav role="navigation" class="row">
+<h2>About this site</h2>
+<section class="col-sm-3">
+<h3>Contact us</h3>
+<ul class="list-unstyled">
+<li><a href="https://github.com/wet-boew/wet-boew/issues/new">Questions or comments?</a></li>
+</ul>
+</section>
+<section class="col-sm-3">
+<h3>About</h3>
+<ul class="list-unstyled">
+<li><a href="../../wet-boew/index-en.html#about">About the Web Experience Toolkit</a></li>
+<li><a rel="external" href="https://www.canada.ca/en/treasury-board-secretariat/topics/government-communications.html">About the Web Standards</a></li>
+</ul>
+</section>
+<section class="col-sm-3">
+<h3>News</h3>
+<ul class="list-unstyled">
+<li><a href="https://github.com/wet-boew/wet-boew-documentation/pulse">Recent project activity</a></li>
+<li><a href="https://github.com/wet-boew/wet-boew-documentation/graphs">Project statistics</a></li>
```

Comment:
href has to be https://github.com/wet-boew/wet-boew/graphs .
Same thing should be done for french version.